src/test/java/teammates/test/cases/browsertests/InstructorFeedbackSubmitPageUiTest.java

```diff
@@ -309,6 +330,66 @@ private void testSubmitAction() throws Exception {
         submitPage.verifyHtmlMainContent("/instructorFeedbackSubmitPageFullyFilled.html");
     }
 
+    private void testAddCommentsToQuestionsWithResponses() throws IOException {
+        ______TS("add new comments on questions with responses and verify add comments without responses action");
+
+        submitPage = loginToInstructorFeedbackSubmitPage("IFSubmitUiT.instr", "Open Session");
+        submitPage.waitForPageToLoad();
+        submitPage.verifyHtmlMainContent("/instructorFeedbackSubmitPageNoComments.html");
+
+        submitPage.addFeedbackResponseComment("-0-1-6", "New MCQ Comment 1");
+        submitPage.addFeedbackResponseComment("-1-1-6", "New MCQ Comment 2");
+        submitPage.addFeedbackResponseComment("-2-1-6", "New MCQ Comment 3");
+        submitPage.addFeedbackResponseComment("-0-1-9", "New MCQ Comment 4");
+        submitPage.addFeedbackResponseComment("-0-1-11", "New MCQ Comment 5");
+        submitPage.addFeedbackResponseComment("-0-1-15", "New MCQ Comment 6");
+
+        submitPage.submitWithoutConfirmationEmail();
+        submitPage.waitForTextsForAllStatusMessagesToUserEquals(Const.StatusMessages.FEEDBACK_RESPONSES_SAVED,
+                Const.StatusMessages.FEEDBACK_UNANSWERED_QUESTIONS + "21, 24.");
+    }
+
+    private void testEditCommentsActionAfterAddingComments() throws IOException {
+        ______TS("edit comments on responses and verify added comments action");
+
+        submitPage = loginToInstructorFeedbackSubmitPage("IFSubmitUiT.instr", "Open Session");
+        submitPage.waitForPageToLoad();
+        submitPage.verifyHtmlMainContent("/instructorFeedbackSubmitPageAddedCommentOnResponses.html");
```

Comment:
In `InstructorFeedbackSubmitPageUiTest.java`, I think after edition verifying wouldn't be possible because after submission, page gets redirected to Home Page.

Reply:
You can always get it back by `getFeedbackEditPage`